frame/nomination-pools/src/tests.rs

```diff
@@ -90,100 +91,104 @@ mod bonded_pool {
-		assert_eq!(bonded_pool.balance_to_point(0), 0);
-
-		// 2 points : 1 balance ratio
-		StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 50);
-		assert_eq!(bonded_pool.balance_to_point(10), 20);
-
-		// 1 points : 2 balance ratio
-		StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 100);
-		bonded_pool.points = 50;
-		assert_eq!(bonded_pool.balance_to_point(10), 5);
-
-		// 100 points : 0 balance ratio
-		StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 0);
-		bonded_pool.points = 100;
-		assert_eq!(bonded_pool.balance_to_point(10), 100 * 10);
-
-		// 0 points : 100 balance
-		StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 100);
-		bonded_pool.points = 100;
-		assert_eq!(bonded_pool.balance_to_point(10), 10);
-
-		// 10 points : 3 balance ratio
-		StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 30);
-		assert_eq!(bonded_pool.balance_to_point(10), 33);
-
-		// 2 points : 3 balance ratio
-		StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 300);
-		bonded_pool.points = 200;
-		assert_eq!(bonded_pool.balance_to_point(10), 6);
-
-		// 4 points : 9 balance ratio
-		StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 900);
-		bonded_pool.points = 400;
-		assert_eq!(bonded_pool.balance_to_point(90), 40);
-	}
+			// 1 points : 1 balance ratio
+			StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 100);
+			assert_eq!(bonded_pool.balance_to_point(10), 10);
+			assert_eq!(bonded_pool.balance_to_point(0), 0);
 
-	#[test]
-	fn balance_to_unbond_works() {
-		// 1 balance : 1 points ratio
-		let mut bonded_pool = BondedPool::<Runtime> {
-			id: 123123,
-			inner: BondedPoolInner {
-				state: PoolState::Open,
-				points: 100,
-				member_counter: 1,
-				roles: DEFAULT_ROLES,
-			},
-		};
+			// 2 points : 1 balance ratio
+			StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 50);
+			assert_eq!(bonded_pool.balance_to_point(10), 20);
 
-		StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 100);
-		assert_eq!(bonded_pool.points_to_balance(10), 10);
-		assert_eq!(bonded_pool.points_to_balance(0), 0);
+			// 1 points : 2 balance ratio
+			StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 100);
+			bonded_pool.points = 50;
+			assert_eq!(bonded_pool.balance_to_point(10), 5);
 
-		// 2 balance : 1 points ratio
-		bonded_pool.points = 50;
-		assert_eq!(bonded_pool.points_to_balance(10), 20);
+			// 100 points : 0 balance ratio
+			StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 0);
+			bonded_pool.points = 100;
+			assert_eq!(bonded_pool.balance_to_point(10), 100 * 10);
 
-		// 100 balance : 0 points ratio
-		StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 0);
-		bonded_pool.points = 0;
-		assert_eq!(bonded_pool.points_to_balance(10), 0);
+			// 0 points : 100 balance
+			StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 100);
+			bonded_pool.points = 100;
```

Comment:
Should be `0` here?

Reply:
hmm let me check, I haven't changed these, only indented them :))

Reply:
640ec312d86f50783740d1c7bf28755442dfa6e4